villageline/settings/local.py.example

```diff
@@ -0,0 +1,8 @@
+from settings import *
```

Comment:
Should this be:

```suggestion
from villageline.settings import *
```

?

I had to make this change otherwise I got:

```
  File "/Users/gmarkall/work/communityline/communityline/villageline/settings/local.py", line 1, in <module>
    from settings import *
ModuleNotFoundError: No module named 'settings'
```

when running

```
python manage.py runserver --settings=villageline.settings.local
```

Reply:
After some more experimenting, I think maybe this should actually have been

```suggestion
from villageline.settings.settings import *
```

Reply:
https://github.com/richardnpaul/communityline/blob/bd30512d362d6f78b2526fb2f9f35a3f9370a475/villageline/settings/settings.py#L23